lib/pltr/v2/store/initialState.js

```diff
@@ -70,6 +70,11 @@ export const ui = {
   timeline: {
     size: 'large',
   },
+  toastProps: {
```

Comment:
Your choice of where to put this state is logically sound; however, the `ui` key gets saved to files.
Could you please put it on a different reducer?
Our system reducers (state that isn't saved to file) are:
 - 'error',
 - 'permission',
 - 'project',
 - 'actions',
 - 'client',
 - 'license',
 - 'knownFiles',
 - 'templates',
 - 'settings',
 - 'backups',
 - 'applicationState', and
 - 'imageCache'.
(As copy-pasted from `systemReducers.js`).

If none of these are a good place, then please feel free to add another reducer (`notifications` perhaps?).